reference/configuration/framework.rst

```diff
@@ -144,6 +144,22 @@ which the cache can serve a stale response when an error is encountered
 (default: 60). This setting is overridden by the stale-if-error HTTP
 Cache-Control extension (see RFC 5861).
 
+terminate_on_cache_hit
```

Comment:
I think we made a mistake by placing this (and the other HTTP cache) options here. If I do not completely miss anything these options cannot be configured using the semantic configuration.

Reply:
Since https://github.com/symfony/symfony/pull/37351 , we do allow the options to be configured through semantic configuration.

I've created https://github.com/symfony/symfony/pull/47630 to also add this new option to it.